package.json

```diff
@@ -14,7 +14,8 @@
     "test": "grunt test",
     "regression-test": "wdio wdio.conf.js",
     "lint": "./node_modules/.bin/eslint .",
-    "lint-auto-fix": "./node_modules/.bin/eslint . --fix"
+    "lint-auto-fix": "./node_modules/.bin/eslint . --fix",
+    "clientBuild": "grunt clientBuild"
   },
   "repository": {
```

Comment:
can you move all grunt dependencies from dev to prod .

"grunt-terser", i have added this to test minify feature but forgot to move to prod dependencies .
Thanks